drivers/flash/flash_mcux_flexspi_hyperflash.c

```diff
@@ -0,0 +1,672 @@
+/*
+ * Copyright (c) 2021 Volvo Construction Equipment
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_imx_flexspi_hyperflash
+
+#include <zephyr.h>
+#include <errno.h>
+#include <drivers/flash.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(flexspi_hyperflash, CONFIG_FLASH_LOG_LEVEL);
+
+#ifdef CONFIG_HAS_MCUX_CACHE
+#include <fsl_cache.h>
+#endif
+
+#include <sys/util.h>
+
+#include "memc_mcux_flexspi.h"
+
+#define SPI_HYPERFLASH_SECTOR_SIZE              (0x40000U)
+#define SPI_HYPERFLASH_PAGE_SIZE                (512U)
+
+#define HYPERFLASH_WRITE_SIZE                   (16)
+#define HYPERFLASH_ERASE_VALUE                  (0xFF)
+
+#ifdef CONFIG_FLASH_MCUX_FLEXSPI_HYPERFLASH_WRITE_BUFFER
+static uint8_t hyperflash_write_buf[SPI_HYPERFLASH_PAGE_SIZE];
+#endif
+
+
+enum {
+	/* Instructions matching with XIP layout */
+	READ_DATA = 0,
+	WRITE_DATA = 1,
+	READ_STATUS = 2,
+	WRITE_ENABLE = 4,
+	ERASE_SECTOR = 6,
+	PAGE_PROGRAM = 10,
+	ERASE_CHIP = 12,
+};
+
+#define CUSTOM_LUT_LENGTH                      64
+
+static const uint32_t flash_flexspi_hyperflash_lut[CUSTOM_LUT_LENGTH] = {
+	/* Read Data */
+	[4 * READ_DATA] =
+		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_DDR, kFLEXSPI_8PAD, 0xA0,
+				kFLEXSPI_Command_RADDR_DDR, kFLEXSPI_8PAD, 0x18),
```

Comment:
Please align the arguments, for example:
```suggestion
		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_DDR,       kFLEXSPI_8PAD, 0xA0,
				kFLEXSPI_Command_RADDR_DDR, kFLEXSPI_8PAD, 0x18),
```